### ISSUE TITLE
Note for writing palette LUTs

### ISSUE THREAD

**Issue by dgobbi** [dgobbi/vtk-dicom#114]
So far, there is nothing in vtk-dicom for generating palette LUT data for writing.  However, it is worth noting that the safe approach, for the sake of maximum compatibility with other DICOM display software, is to use 16 bits per table entry, and to store the same value in the lower 8 bits as in the high 8 bits.  In this way, buggy software that reads just the low byte, or that gets the endianness wrong, will still display the correct color table.

The 16 bit data would be the same as 8 bit data, but multiplied by 0x0101 (or 257).  Note that this is exactly equal to 65535/255, so there is no approximation involved.  Multiplying by 257 converts the 8-bit value to the corresponding 16-bit value.

In other words, I should not attempt to use "true" 16-bit palette LUTs.  I should only use 8-bit palette LUTs, and convert them to 16 bit only when saving them in the DICOM file.

I'm closing this immediately, since it is merely informative and doesn't require any changes to the existing code.
